core_backend/app/auth/dependencies.py

```diff
@@ -99,36 +84,31 @@ def get_current_user(
         if username is None:
             raise credentials_exception
 
-        user = AuthenticatedUser(
-            username=username,
-            access_level=cast(AccessLevel, USERS[username]["access_level"]),
-        )
-
+        # fetch user from database
+        async with AsyncSession(
+            get_sqlalchemy_async_engine(), expire_on_commit=False
+        ) as asession:
+            try:
+                user_db = await get_user_by_username(username, asession)
+                await asession.aclose()
```

Comment:
Don't need close anymore. The context manager sorts this out

Reply:
Woops, will clean these up